style.css

```diff
@@ -26,10 +26,65 @@
 }
 
 .menu {
+  background-color: #3c3a39;
   position: absolute;
-  top: 55px;
-  left: 330px;
+  top: 45px;
+  left: 300px;
   margin-top: -30px;
+  border: none;
+}
```

Comment:
* Kindly ensure that your menu drop-down fits and is responsive as it currently has a horizontal scroll which gives it not so good user experience below is a recording of the issue 

https://user-images.githubusercontent.com/52497246/208438086-8d298f91-dfef-4493-8d7f-9cc5b0f22f45.mp4